app/serializers/gobierto_people/person_serializer.rb

```diff
@@ -28,11 +34,43 @@ def political_group
     end
 
     def position
-      object.charge
+      filtered_positions.first&.name
+    end
+
+    def all_positions_html
+      object.historical_charges.reverse_sorted.join("<br>")
+    end
+
+    def filtered_positions
```

Comment:
memoize please, it's used in many methods

Reply:
Totally agree.